zebra/interface.c

```diff
@@ -4853,17 +4853,14 @@ static int ag_iter_cb(const struct lyd_node *dnode, void *arg)
 void cli_show_legacy_admin_group(struct vty *vty, const struct lyd_node *dnode,
 				 bool show_defaults)
 {
-	if (!yang_dnode_exists(dnode, "./legacy-admin-group"))
-		return;
-
 	vty_out(vty, "  admin-group 0x%x\n",
```

Comment:
hmm - so since the actual config syntax is "admin-grp" this ... doesn't work either?